onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -1593,16 +1593,44 @@ def aten_upsample_bicubic2d_backward(
     raise NotImplementedError()
 
 
+@torch_op("aten::upsample_bilinear2d", trace_only=True)
 def aten_upsample_bilinear2d(
     self: TensorType,
-    output_size: INT64,
+    size: INT64,
```

Comment:
I would match this with the aten signature below. We don't need to handle cases when output_size is None or align_corners is None because they are not defined in the signature. For the `scales_h=None, float? scales_w=None` is not None case I suggest creating an overload until `Optional` for attributes in supported in ONNX.